data/presets.yaml

```diff
@@ -124,7 +124,8 @@ parsers:
 - sqlite/chrome_66_cookies
 - sqlite/chrome_autofill
 - sqlite/chrome_extension_activity
-- sqlite/firefox_cookies
+- sqlite/firefox_2_cookies
+- sqlite/firefox_10_cookies
```

Comment:
maybe change 10 to match the firefox version in which the change was made

Reply:
nevermind I see https://searchfox.org/mozilla-central/source/netwerk/cookie/CookiePersistentStorage.cpp#1298 the schema has a version, that is fine as well